travis/travis_install_nightly

```diff
@@ -3,7 +3,7 @@
 set -e
 
 if [ "${LINT_CHECK}" != "0" ]; then
-    pip install -q flake8=="3.4.1" Click pylint-mccabe
+    pip install -Uq flake8=="3.4.1" Click pylint-mccabe pbr
```

Comment:
pbr?

Reply:
pylint-odoo is using pbr and I had problems locally with the order of installation like as the last one with saas npm package